### DISCUSSION TITLE
6 suggestions

### DISCUSSION THREAD

**Discussion by tigertank909** [voxeltycoon/issues#850] [UI&UX]
having a new type of road highways 2. sidewalk traffic 3. cargo/passenger airports 4. more then one city in regions 5. more things like uranium, farms, fishing etc 6. being able to build roads side ways like rails

**Comment by 776166** [comment]
>> 5. more things like uranium, farms, fishing etc

https://steamcommunity.com/sharedfiles/filedetails/?id=2458009323&searchtext=uranium
https://steamcommunity.com/sharedfiles/filedetails/?id=2493016093&searchtext=food

Bonus:
https://steamcommunity.com/sharedfiles/filedetails/?id=2462181884
https://steamcommunity.com/sharedfiles/filedetails/?id=2516997684
